focus-block/src/Components/BlockForm/BlockForm.js

```diff
@@ -213,6 +214,8 @@ class BlockForm extends Component {
 								name="customTimer"
 								placeholder="Time in mins"
 								type="number"
+                                min="1"
+								max="525600"
```

Comment:
Can we change max to be 1,440? That should be max of 24 hours I think that is well over the amount of time haha.